common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -69,100 +76,255 @@ hfe_initializer_egypt_main = {
+					give_technology = { message = no tech = tech_cloning }
+					give_technology = { message = no tech = tech_gene_banks }
+					give_technology = { message = no tech = tech_gene_seed_purification }
+					give_technology = { message = no tech = tech_gene_expressions }
+					give_technology = { message = no tech = tech_selected_lineages }
+					give_technology = { message = no tech = tech_glandular_acclimation }
+					give_technology = { message = no tech = tech_genetic_resequencing }
+					# The tools to build and upload Thoth
+					give_technology = { message = no tech = tech_sapient_ai	}
+					give_technology = { message = no tech = tech_synthetic_workers }
+					give_technology = { message = no tech = tech_synthetic_leaders }
+					give_technology = { message = no tech = tech_droid_workers }
+					give_technology = { message = no tech = tech_robotic_workers }
+					give_technology = { message = no tech = tech_robomodding }
+					give_technology = { message = no tech = tech_robomodding_points_1 }
+					give_technology = { message = no tech = tech_robomodding_points_2 }
 					kill_leader = {
 						type = ruler
 						show_notification = no
 					}
 					create_leader = {
-						name = Amonhirkhopshef
+						name = "Amonhirkhopshef Mutatistep"
 						species = event_target:hfe_egypt_species
 						class = ruler
-						skill = 5
+						skill = 10
 						leader_age_min = 250
 						leader_age_max = 650
+#						immortal = yes # Had says it's automatically applied from the species
+						gender = male
 						traits = {
 							trait = leader_trait_ruler_chosen
 							trait = hfe_ruler_trait_transcendent_0
 						}
 					}
 					assign_leader = last_created_leader
-					while = {
-						limit = {
-							exists = ruler
-							ruler = {
-								gender = female
-							}
-						}
-						kill_leader = {
-							type = ruler
-							show_notification = no
-						}
-						create_leader = {
-							name = Amonhirkhopshef
-							species = event_target:hfe_egypt_species
-							class = ruler
-							skill = 5
-							leader_age_min = 250
-							leader_age_max = 650
-							traits = {
-								trait = leader_trait_ruler_chosen
-								trait = hfe_ruler_trait_transcendent_0
-							}
-						}
-						assign_leader = last_created_leader
+#					# Must initialize global designs here
+					add_global_ship_design = "NAME_Reaper" # Colossus!
+					add_global_ship_design = "NAME_Imperium"
+					add_global_ship_design = "NAME_Supremacy"
+					add_global_ship_design = "NAME_Glory"
+					add_global_ship_design = "NAME_Devastator"
+					add_global_ship_design = "NAME_Servitor"
+					add_global_ship_design = "NAME_Destiny"	
+					add_global_ship_design = "NAME_Protector"
+					if = {
+						limit = { has_global_flag = HFE_TSP_spawn_test }
+						log = "Ju Ahet actual country installed"
 					}
-					save_global_event_target_as = hfe_egypt_country
+					add_modifier = {
+						modifier = hfe_navy_size
+						days = -1
```

Comment:
no infinite cheat modifiers

Reply:
Without it, the Ju Ahets are miles over their Fleet Limit. Did you ever think that I changed it for a reason?